docs/guides/implementing-connectors.md

```diff
@@ -0,0 +1,88 @@
+---
+id: implementing-connectors
+title: Implement connectors
+description: "Connectors communicate with any system or technology, reducing the time it takes to automate and orchestrate business processes across systems."
+keywords: [connector, modeling, connectors]
+---
+
+The launch of [Camunda Platform 8](../components/concepts/what-is-camunda-platform-8.md) also introduced an integration framework with a key goal: integrate faster to reduce the time it takes to automate and orchestrate business processes across systems.
+
+This goal is achieved using [connectors](../components/modeler/web-modeler/connectors/index.md). Ready to use out of the box, connectors help automate complex [business processes](../components/concepts/processes.md) by inserting them into [BPMN diagrams](./automating-a-process-using-bpmn.md) within [Web Modeler](../components/modeler/about.md), and configuring them via the properties panel.
+
+Connectors technically consist of two parts: the business logic is implemented as a job worker, and the user interface during modeling is provided using an element template. In this guide, we'll walk step-by-step through the implementation of a sample connector.
+
+## Set up
+
+We'll implement our connector with [Modeler](../components/modeler/about.md). To get started, ensure you’ve [created a Camunda Platform 8 account](./guides/create-account.md).
+
+### Create a cluster
+
+import CreateCluster from './assets/react-components/create-cluster.md'
+
+## Getting started
+
+Once logged in to your Camunda Platform 8 account, take the following steps:
+
+1. Click on the **Modeler** tab at the top of the page, and click **New project**.
+2. Name your project. In this example, we'll name ours `Expense process`.
+3. Select **New > BPMN Diagram > + Create blank**.
+4. Give your model a descriptive name, and then give your model a descriptive id within the **General** tab inside the properties panel on the right side of the screen. In this case, we've named our model `Submit expense` with an id of `submitting-expense`.
+
+## Build a BPMN diagram
+
+Use Web Modeler to design a BPMN flow with the appropriate tasks. To get started, create a task by dragging the rectangular task icon from the palette, or click the existing start event and the displayed task icon.
+
+In this example, we've designed the following BPMN diagram:
+
+![bpmn example diagram](./img/bpmn-expense-sample.png)
```

Comment:
[This is outside the scope of this PR.]

Seeing this diagram I realize we'll need to have guidance around alt text for BPMN diagrams to meet accessibility standards, but also make this content available for low-bandwidth geos if the image doesn't load (although with heavy JS, the whole site may not load). In a quick Google search, I can see guidance for diagrams in general, but I didn't quickly find BPMN guidance, which could have a number of branches and end events (unlike a flow chart, for example). 

Can you create an issue to address creating guidance for alt text for BPMN diagrams? It will include research, seeing if OMG has anything they recommend, and even working with web marketing to see what they have planned. This will become more urgent and critical if we prioritize accessibility, but for now my concern is how users would or could follow building or using this diagram if they couldn't see the image. Maybe the "quick" solution is to link to the BPMN instead of building it.

Reply:
https://github.com/camunda/camunda-platform-docs/issues/1046